web/beacon-app/cypress/tsconfig.json

```diff
@@ -10,8 +10,9 @@
     "strict": true,
     "module": "esnext",
     "noImplicitAny": true,
-    "resolveJsonModule": true
+    "resolveJsonModule": true,
+    "types": ["cypress", "@testing-library/cypress"],
   },
-  "types": ["cypress", "@testing-library/cypress"],
+  "include": ["**/*.ts"],
```

Comment:
I kept getting an error message stating that `cy` couldn't be found, so that's why these small updates were made.